### PR TITLE
Use page-specific override files instead of work-overrides.styl

### PR DESCRIPTION
**Pull request by kenworthyc**
### Decentralizing overrides for specific case studies
This PR moves case-study-specific styles from `/src/stylus/work-overrides.styl` (a file which has been growing longer and longer in the tooth) to individual files in this pattern: `/src/stylus/work/*-separate.styl`. Separating styles per case study will make it far easier to remove retired styles in the future.

Moving forward, v3 case studies load the scaffolding file `/src/stylus/work-v3-separate.styl` loaded in the `head` tag (`templates/partials/head.html`), and additional styles (if necessary) load _after_ the content, just before the scripts in the foot (`templates/partials/foot.html`).

### Steps to hook up a case study to a new style override file:

1. Add any styles into a file matching this format:
  `/src/stylus/work/sample-slug-separate.styl`
2. Add any necessary color variables to `/src/stylus/variables.styl`
3. Add this property at the root of the case study:
  `styleOverride: work/sample-slug-separate`

The trailing `*-separate` in the filename is necessary.

Quiet Life is the only remaining case study designed in the v2 style, so the `work-v2-separate.styl` file has been imported into its styleOverride file.

### Corollary changes
- Renamed style bootstrapper: `style.styl` => `index.styl`, because why was it even called that anyway
- All variables have been moved color to `/src/stylus/variables.styl`
- `variables`, `nib`, `sop-styl`, and `mixins` are being imported via `gulp-stylus` rather than via the `index.styl`
  Relevant changes can be found in the `style`, `style-dist`, and `style-stage` tasks. These variables and tools are now more globally available, and further decentralization will be simpler in the future.
- Most `next-case-study` buttons have been fixed so the entire loop is preserved
- Whitespace between sections and items have been added in certain case study yamls for readability
- Unused `gallery.styl` and `outdated-browsers.styl` have been murdered
- Classes `work-section-v3__logo` and `work-section-v3__logo--small` have been reworked for more responsiveness
  Not ideal, but better than before– logos don't render too large on tablet-size screens.

### CODE REVIEW

File: src/stylus/work/av-v2-separate.styl [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        padding 0 auto
+        width 100%
+
+    header.work-section:first-of-type
+      padding 0
+
+    section.work-section:first-of-type
+      padding 0 auto 50px
+
+      @media screen and (max-width 1000px)
```
Comment (krispy1298) [inline src/stylus/work/av-v2-separate.styl]: should probably make a var for 1000px

File: src/stylus/variables.styl [hunk 2]
```diff
@@ -37,3 +39,43 @@ $serif
...
+$blue-chill ?= #0fa399
+$bright-turquoise ?= #14ded0
+$cerulean ?= #0199d9
+$fun-blue ?= #1c449c
+$prussian-blue ?= #002950
+$downriver ?= #072643
+$dark-blue ?= #00274d
+
+$jacarta ?= #322d70
+$violet ?= #1a0731
```
Comment (krispy1298) [inline src/stylus/variables.styl]: I don't love the idea of mucking up the variables file with, what is essentially, one-off color vars for case studies. can the case study colors be either a separate file that you import at the end of variables or moved to their respective separate files? 

Thoughts?

Comment (kenworthyc) [inline src/stylus/variables.styl]: I agree, it'll get pretty hectic in that file pretty soon. Let's have another file for case study colors.

File: src/stylus/variables.styl [hunk 3]
```diff
@@ -39,43 +38,3 @@ $serif
 
 // NOTE: DO NOT use for fonts to avoid cross-origin issues
 $assets-url ?= '/assets/'
```
Comment (krispy1298) [inline src/stylus/variables.styl]: @kenworthyc need to import color-vars here right?

Comment (kenworthyc) [inline src/stylus/variables.styl]: ah yep, good call

File: src/templates/work/quiet-life.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {{>head}}
-  {{>workTopHeaderV3}}
+  {{>workTopHeaderV2}}
```
Comment (kenworthyc) [inline src/templates/work/quiet-life.html]: I'd thought v3 styles weren't being loaded for QL, but I was mistaken. fixed up in `bd74927`.

File: src/stylus/work/cet-v2-separate.styl [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        background-color $white
+
+      &-brand-book
+        .work-section &
+          @media screen and (min-width 901px)
+            width 50%
+            max-width 750px
+            margin 3em 0 1em!important
+
+          @media (max-width 900px)
```
Comment (kenworthyc) [inline src/stylus/work/cet-v2-separate.styl]: replaced all these in 51eaaa5

File: src/stylus/work/second-city-v2-separate.styl [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+      @media (max-width: $tablet-size-break)
+        display inline-block
+        width 100%
+
+    .svg-1,
+    .svg-2
+      display block
+      width 95%
+
+      @media (max-width: 900px)
```
Comment (kenworthyc) [inline src/stylus/work/second-city-v2-separate.styl]: killed in 51eaaa5

Comment (kenworthyc) [inline src/stylus/work/second-city-v2-separate.styl]: god bless github's automatic commit links

**Pull request merged**